packages/sdk/src/services/editLp/editLp.ts

```diff
@@ -0,0 +1,194 @@
+import { BigNumber, ContractReceipt } from 'ethers';
+import {
+  estimateGas,
+  executeTransaction,
+  simulateTx,
+} from '../executeTransaction';
+import {
+  getNativeGasToken,
+  convertGasUnitsToNativeTokenUnits,
+} from '@voltz-protocol/sdk-v1-stateless';
+import { getTokenDetails, scale, descale } from '@voltz-protocol/commons-v2';
+import { notionalToLiquidityBN } from '../../utils/helpers';
+import { defaultAbiCoder } from 'ethers/lib/utils';
+import { CompleteEditLpDetails, LpArgs, LpPeripheryParameters } from './types';
+import { getEditLpPeripheryParams } from './getEditLpPeripheryParams';
+import { InfoPostLp } from '../lp';
+import { encodeLp } from '../lp/encode';
+
+async function createLpParams({
+  poolId,
```

Comment:
think if we have access to positionId, then we should be able to derive the poolId from the positionId (see v1 sdk editSwap as an example, wdyt?

Reply:
true, I can just have position Id